Description: Reproducible version hash table.
 - For version#"configure arguments", we replace the build directory
   with "/build/path" in all the instances of the -fdebug-prefix-map
   flag.
 - For version#"compile-node-name", we use a dummy value.
Author: Doug Torrance <dtorrance@piedmont.edu>
Forwarded: not-needed
Last-Update: 2021-09-23

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -6,7 +6,7 @@
 AC_SUBST(CONFIGURED_FILES,"$ac_config_files")
 AC_SUBST(CONFIG_ARGS,"$ac_configure_args")
 echo "'$0' $ac_configure_args" > config.args
-C_CONFIG_ARGS=` echo "$ac_configure_args" | sed -e 's=\\\\=\\\\\\\\=g' -e 's=\\"=\\\\"=g' `
+C_CONFIG_ARGS=`echo "$ac_configure_args" | sed -e 's=\\\\=\\\\\\\\=g' -e 's=\\"=\\\\"=g' -e "s=$CURDIR=/build/path=g"`
 AC_DEFINE_UNQUOTED(CONFIG_ARGS,"$C_CONFIG_ARGS",arguments used for configure)
 AC_SUBST(CONFIG_CMD,"'$0' $ac_configure_args")
 
@@ -73,7 +73,7 @@
 dnl       $ eval $A
 dnl       hi
 
-AC_SUBST(NODENAME,"`uname -n`")
+AC_SUBST(NODENAME,"m2-compile-node")
 AC_DEFINE_UNQUOTED(NODENAME,"$NODENAME",hostname used for compilation)
 
 AC_SUBST(ISSUE) # no initial value
